modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/bottomsheet/BottomSheetContent.kt

```diff
@@ -0,0 +1,143 @@
+package au.com.shiftyjelly.pocketcasts.compose.bottomsheet
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppThemeWithBackground
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH40
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP50
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+
+private val outlinedBorder: BorderStroke
+    @Composable
+    get() = BorderStroke(2.dp, MaterialTheme.colors.primary)
+
+class BottomSheetContentState(
+    val content: Content,
+) {
+    data class Content(
+        val titleText: String,
+        val summaryText: String,
+        val primaryButton: Button.Primary,
+        val secondaryButton: Button.Secondary? = null,
+    ) {
+        sealed interface Button {
+            val label: String
+            val onClick: (() -> Unit)?
+
+            data class Primary(
+                override val label: String,
+                override val onClick: () -> Unit,
+            ) : Button
+
+            data class Secondary(
+                override val label: String,
+                override val onClick: (() -> Unit)? = null,
```

Comment:
In the current implementation, [clicking the secondary button dismisses the bottom sheet](https://github.com/Automattic/pocket-casts-android/pull/413/files#diff-7039a52beb217dffc340f844cd92bd2dd2d3a21a7e228981b82fbb0def9ed2dfR109-R110). If that's the only behavior we need, we might not want to set any click listener. 

I'm open to making any changes in case we need to.